onnx/funconnx/op_run.py

```diff
@@ -0,0 +1,508 @@
+# SPDX-License-Identifier: Apache-2.0
+# pylint: disable=C0415,R0912
+
+from abc import ABC, abstractmethod
+from typing import Any, Dict, Iterable, List, Optional, Tuple
+
+import numpy as np  # type: ignore
+
+from ..defs import get_all_schemas_with_history, get_schema, onnx_opset_version
+from ..helper import make_node
+from ..numpy_helper import to_array
+from ..onnx_pb import AttributeProto, GraphProto, NodeProto, TypeProto
+
+
+class RuntimeTypeError(RuntimeError):
+    """
+    Raised when a type of a variable is unexpected.
+    """
+
+
+class DefaultNone:
+    """
+    Default value for parameters when the parameter is not set
+    but the operator has a default behavior for it.
+    """
+
+
+class RefAttrName:
+    """
+    Implements a link between a parameter of a function
+    and an attribute in node.
+
+    :param name: name of the input
+    """
+
+    def __init__(self, name: str):
+        self.name = name
+
+    def __repr__(self) -> str:
+        "usual"
+        return f"{self.__class__.__name__}({self.name!r})"
+
+
+def _build_schemas() -> Dict[str, type]:
+    res: Dict[str, type] = {}
+    for schema in get_all_schemas_with_history():
```

Comment:
Seems to ignore schema.domain? Should either use (domain, opname) or filter out only ops belonging to standard domain. Not sure which is intended.

Reply:
This no collision of operator name accross domains but I should fix it.